packages/rancher-gatekeeper/package.yaml

```diff
@@ -1,6 +1,6 @@
-url: https://open-policy-agent.github.io/gatekeeper/charts/gatekeeper-3.2.1.tgz
+url: https://open-policy-agent.github.io/gatekeeper/charts/gatekeeper-3.3.0.tgz
 packageVersion: 02
-releaseCandidateVersion: 00
+releaseCandidateVersion: 01
```

Comment:
Have I done this version change correctly?

Reply:
This should happen once per PR so that it creates a new version, so yes it looks like it.